client/my-sites/backup/rewind-flow/granular-restore.tsx

```diff
@@ -39,12 +41,15 @@ interface Props {
 	siteUrl: string;
 }
 
+type BackupGrandularRestoreType = 'file' | 'theme' | 'plugin' | 'table';
```

Comment:
Maybe we could move this type definition to `client/state/rewind/browser/types.ts` and use it in `BackupBrowserItem` and `BackupBrowserSelectedItem` as well. What do you think?

Reply:
Yeah I'll move it over there and rename it.  Also the naming is wrong.  Grandular 😓 I have high hopes for these restores.